global.d.ts

```diff
@@ -1,3 +1,5 @@
-export {}; // eslint-disable-line
+declare module "*.svelte" {
+  export { SvelteComponentDev as default } from "svelte/internal";
+}
```

Comment:
this is part of a fix that caused some warnings during imports about missing types